json_test.go

```diff
@@ -43,7 +43,7 @@ func TestMarshalJSON(t *testing.T) {
 		{&oneExportedField{A: "Z"}, `{"A":"Z"}`, ""},         // #6
 		{[]string{"a", "bc"}, `["a","bc"]`, ""},              // #7
 		{[]interface{}{"a", "bc", 10, 10.93, 1e3},
-			``, "Unregistered"}, // #8
+			``, "unregistered"}, // #8
```

Comment:
what about this place?

Unregistered == unregistered interface interface {}?

Reply:
This is correct because the error is now lowercase and not capitalized